components/tfx/README.md

```diff
@@ -0,0 +1,17 @@
+## TFX components for Kubefow Pipelines
+
```

Comment:
May I say this folder contains KFP Components based on TFX Component?

Aka it's for users who use KFP SDK but want to use TFX Component.

We may need to detail this msg.

Reply:
on the other hand, for compatibility with uDSL, I think reuse KFP component inside TFX SDK has higher priority, right? feel free to correct me, I'm not in the loop for compatibility discussion.

Reply:
> Aka it's for users who use KFP SDK but want to use TFX Component.
> 
> We may need to detail this msg.

I have the following sentences below: "These components were created to allow the users to use TFX components in their KFP pipelines, to be able to mix KFP and TFX components."

How would you change the wording to make it easier to understand?

Reply:
I'm wondering whether we explictlly mention the SDK. These components are only for KFP SDK. 

(not a big deal, as the format is argo-styled, user can guess out it's for KFP SDK). 


These components were created to allow the users to use TFX components in their KFP pipelines **made by KFP SDK**

How about mention KFP pipelines **made by KFP SDK**?

Reply:
> How about mention KFP pipelines **made by KFP SDK**?

"Versions of TFX components that can be used with KFP SDK". WDYT?